src/test/scala-2.11/org/apache/james/gatling/smtp/scenari/FeederNoAuthenticationNoEncryptionScenario.scala

```diff
@@ -0,0 +1,25 @@
+package org.apache.james.gatling.smtp.scenari
+
+import io.gatling.core.Predef._
+import org.apache.james.gatling.control._
+import org.apache.james.gatling.jmap.scenari.common.Configuration._
+import org.apache.james.gatling.smtp.scenari.common.Configuration._
+
+import scala.concurrent.duration._
+
+class FeederNoAuthenticationNoEncryptionScenario extends Simulation {
+
+  def feeder = csv("users.csv")
+
+  val scn = scenario("SMTP_No_Authentication_No_Encryption")
+    .feed(feeder)
```

Comment:
why do you need to copy/paste the scenario just to change the feeder ?

Reply:
1/ I think it's cool to keep the main steps in the scenario; that way you are able to understand what part of the server will be stressed while playing it.

2/ It's only 10 lines of code.